packages/Modals/index.ts

```diff
@@ -1,20 +1,23 @@
-import { App, Plugin } from 'vue';
```

Comment:
Why are you changed to to double quote. I thought we are using single quotes everywhere,

Reply:
I think, there was no rule. I've added `.prettierrc` file and changed to single quotes.